gemrb/core/GUI/GameControl.cpp

```diff
@@ -2038,6 +2028,33 @@ bool GameControl::HandleActiveRegion(InfoPoint *trap, Actor * actor, const Point
 	return false;
 }
 
+// Calculate the angle between a clicked point and the first selected character,
+// so that we can set a sensible orientation for the formation
+void GameControl::InitFormation (const Point &clickPoint)
+{
+	/* We get called on initial click, but also on drag.  Perform the setup only once.  */
+	if (isFormationRotation)
+		return;
+
+	if (core->GetGame()->selected.size() == 0) {
+		isFormationRotation = false;
+		return;
+	}
+
+	isFormationRotation = true;
+	Game* game = core->GetGame();
```

Comment:
```suggestion
	Game *game = core->GetGame();
```

Reply:
I can change that, but there is absolutely no consistent style in this file, and this is just copied from another location.

Reply:
This is the style from most of the project, so the most consistent you can get. It will be enforced soon (still working on #161).